cleanlab/count.py

```diff
@@ -202,26 +194,22 @@ def calibrate_confident_joint(confident_joint, labels, *, multi_label=False) ->
       An array of shape ``(K, K)`` representing the confident joint, the matrix used for identifying label issues, which
       estimates a confident subset of the joint distribution of the noisy and true labels, ``P_{noisy label, true label}``.
       Entry ``(j, k)`` in the matrix is the number of examples confidently counted into the pair of ``(noisy label=j, true label=k)`` classes.
+      This should be in same format expected by :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` function.
```

Comment:
Is the format listed in cleanlab.find_label_issues identical to the description here? Suggesting to either remove some of this description and link more information on the format or remove the link as it does not provide additional information.

Reply:
deleted link to filter.find_label_issues and mention of it. I just included that to reassure user they don't need to change anything from that format (they may already be familiar with), but I don't think this reassurance is critical